randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -495,6 +495,8 @@ Extra - minimap_data: [{'x': 64, 'y': 22}, {'x': 64, 'y': 23}, {'x': 64, 'y': 24
 > Horizontal Dock to Spazer Beam Chamber Access; Heals? False
   * Layers: default
   * 4-Tiles High Dock to Spazer Beam Chamber Access/Horizontal Dock to Upper Factory Intersection
+  > Dock to Upper Factory Save Station
+      Speed Booster and Shinesparking Tricks (Beginner) and Walljump (Beginner) and Disabled Entrance Rando
```

Comment:
isnt this also possible with PG / HJ / Space rathern than requiring WJ?

Reply:
Could be HJ, space is already covered in the other transitions between room bottom->mid->upper. PG could also help reach the platform to spark straight up to the save.